### PR TITLE
Add KeyStoreUtil

### PR DESCRIPTION
**Pull request by grevend-bosch**
Blocked by: ~~carbynestack/common#18~~ & #75

Signed-off-by: David Greven <fixed-term.David.Greven@de.bosch.com>

### CODE REVIEW

File: src/test/java/io/carbynestack/cli/util/KeyStoreUtilFailuresTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class KeyStoreUtilFailuresTest {
+    @ParameterizedTest
+    @EnumSource(value = KeyStoreUtilFailures.class)
+    void reason(KeyStoreUtilFailures failure) {
+        assertThat(failure.synopsis()).isEqualTo("Failed to generate temporary keystore.");
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = KeyStoreUtilFailures.class)
```
Comment (sbckr) [inline src/test/java/io/carbynestack/cli/util/KeyStoreUtilFailuresTest.java:22]: Is this test necessary as `AbstractCharSequenceAssert::isNotBlank`as  used in test [descriptionBlank](https://github.com/carbynestack/cli-ng/blob/9be637e502c93207164460b1b3c8f33cb07e8aa0/src/test/java/io/carbynestack/cli/util/KeyStoreUtilFailuresTest.java#L29) does also verify that the actual CharSequence is **not** empty (according to the methods documentation)

**Review by sbckr — COMMENTED** [review]
See comments on [src/test/java/io/carbynestack/cli/util/KeyStoreUtilFailuresTest.java](src/test/java/io/carbynestack/cli/util/KeyStoreUtilFailuresTest.java)

**Comment by sbckr** [comment]
@grevend-bosch blocked as tests are failing

**Comment by grevend-bosch** [comment]
> @grevend-bosch blocked as tests are failing

The main reason is that some classes from cs-common do not yet exist in this repository.

**Pull request merged**